emgapianns/management/commands/import_sample.py

```diff
@@ -82,6 +76,14 @@ def fetch_sample_api(accession):
 
     def create_or_update_sample(self, ena_db_model, api_data):
         accession = api_data['secondary_sample_accession']
+        logger.debug('Getting the biome from the DB.')
+        try:
+            biome_model = emg_models.Biome.objects.using(self.emg_db).get(lineage=self.biome)
+        except emg_models.Biome.DoesNotExist as exception:
+            logger.exception(exception)
+            logger.error(f'The supplied biome is not valid. Biome: "{self.biome}"')
+            sys.exit(1)
```

Comment:
We use `sys.exit` instead of the (arguably) more explicit `raise CommandError(1)`.
Is there a reason for that? Maybe because of how it is handled when this command is called from `call_command(..)`?

Reply:
I don't think there is a reason for that, just coding style. We could change all of those sys.exit calls with CommandError, I've added TODOs on them. ATM I don't have the time to fix them.